docs/changelog-configuration.md

```diff
@@ -103,6 +103,17 @@ YAML encoding that is compatible with ansible-lint's default rules.
     The exact format used might be adjusted in the future if new releases of ansible-lint
     adjust their yamllint configuration.
 
+### `changelog_sort` (string)
+
+The default is `alphanumerical`.
+
+This option controls the sorting of changelog entries in `changelogs/changelog.yaml`. It accepts the following values:
+
+- `unsorted`: No sorting is performed on the changelog entries. The entries are stored in the order they were added.
+- `version`: Sorts the changelog entries by semantic version in ascending order.
+- `version_reverse`: Sorts the changelog entries by semantic version in descending order.
```

Comment:
I think it should be `version_reversed`:
```suggestion
- `version_reversed`: Sorts the changelog entries by semantic version in descending order.
```
Also, it does not necessarily use semantic versioning, unless that has been explicitly configured (see `use_semantic_versioning` option) or if antsibull-changelog is used for Ansible collections. So please remove the "semantic" from both descriptions.